install/kubernetes/tetragon/templates/_helpers.tpl

```diff
@@ -82,6 +82,14 @@ ServiceAccounts
 {{- end -}}
 {{- end }}
 
+{{- define "rthooks.serviceAccount" -}}
+{{- if .Values.rthooks.serviceAccount.name -}}
+{{- printf  "%s" .Values.rthooks.serviceAccount.name -}}
+{{- else -}}
+{{- printf  "%s-rthooks-service-account" .Release.Name -}}
```

Comment:
nit: I would say you don't need to repeat "service-account" in the name since it is a service account object already, so it should be pretty obvious.

Reply:
We do it for the other service accounts, but I guess we can do the right thing here. Fixed.